tests/sandbox/create_message_batches/test_field_validation.py

```diff
@@ -154,7 +174,19 @@ def test_data_null(nhsd_apim_proxy_url, property, pointer, correlation_id):
 @pytest.mark.parametrize("correlation_id", CORRELATION_IDS)
 def test_data_invalid(nhsd_apim_proxy_url, property, pointer, correlation_id):
     """
-    .. py:function:: Test invalid values
+    .. py:function:: Scenario: A user submitting a request with an invalid required attribute \
+        in the request body receives a 400 'Invalid Value' response
+
```

Comment:
Needs a list of invalid properties

Reply:
Addressed in request_properties.rst, linked below given/when/then